src/client/delphi_epidata.py

```diff
@@ -136,18 +136,13 @@ def fluview(regions, epiweeks, issues=None, lag=None, auth=None):
         if auth is not None:
             params["auth"] = auth
         # Make the API call
-        return Epidata._request(params)
+        return Epidata._request(params, "fluview")
```

Comment:
```suggestion
        return Epidata._request("fluview", params)
```

here and the other endpoints